kubernetes/internal/voyager-ingress-template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: voyager.appscode.com/v1beta1
+kind: Ingress
+metadata:
+  name: voyager-ingress
+  namespace: default
```

Comment:
I thought this was in the voyager namespace?  Also, does this have to go in the namespace of the domain, which might not be default.

Reply:
fixed. this is the namespace of Voyager Ingress resource which is the same namespace of domain.